modules/tinymce/src/models/dom/test/ts/module/table/TableTestUtils.ts

```diff
@@ -132,12 +135,15 @@ const pDragResizeBar = async (editor: Editor, rowOrCol: 'row' | 'column', index:
   const resizeBar = await Waiter.pTryUntil('wait for resize bars',
     () => UiFinder.findIn(docElem, `div[data-${rowOrCol}='${index}']`).getOrDie()
   );
+  // a real user won't drag within a single frame, so we shouldn't either
   Mouse.mouseDown(resizeBar);
+  await Waiter.pWait(0);
```

Comment:
Why then why are these not 17ms, 0 ms is not a frame either. But adding that would slow things down.